tests/python/unittest/test_operator.py

```diff
@@ -91,6 +91,24 @@ def test_lstm_bidirectional():
 
     check_rnn_consistency(stack, fused, T, N, I, H)
 
+# Currently, fused LSTM operator doesn't support dropout.
+# Will change this test after dropout is supported
+@with_seed()
+def test_lstm_dropout():
+    X = mx.sym.Variable('x')
+    Params = mx.sym.Variable('params')
+    HX = mx.sym.Variable('state')
+    CX = mx.sym.Variable('state_cell')
+    T, N, I, H = 300, 20, 800, 800
+    rnn = mx.sym.RNN(data=X, parameters=Params, state=HX, state_cell=CX,
+                     state_size=H, num_layers=5, mode='lstm', p=0.5, state_outputs=True, name='LSTM')
+    exe = rnn.simple_bind(ctx=mx.cpu(), x=(T, N, I))
+    try:
+        out = exe.forward(is_train=False)
+        out[0].wait_to_read()
+        assert False  # should not reach here
+    except mx.base.MXNetError as err:
```

Comment:
Excellent approach! This will ensure we don't miss it to re-enable the test when we introduce dropout. Great job

Reply:
Yes. Also to ensure the failure happens at a proper position and correct error message is presented. Follow @reminisce 's idea in #10844 .